### ISSUE TITLE
You missed an f-string

### ISSUE THREAD

**Issue by rmanthorpe** [John-Colvin/YAIDIP#5]
Line 469: `assert(y > 42, format(f"y is $y, should be greater than 42"));`

**Comment by schveiguy** [comment]
In fact, f strings don't seem to exist anywhere except one spot. I think they were removed (a decision I agree with). However, there is still mention in the text about handling both styles, which should also be removed.

**Comment by rmanthorpe** [comment]
Yeah that's what I meant - @andralex missed one when removing the f-string syntax

**Comment by schveiguy** [comment]
oof, totally misread your comment! Yes, and there are more mentions of format strings that need addressing (maybe separate issue though).

**Comment by rmanthorpe** [comment]
You're much more observant than I am! I only spotted the one!
